polkadot/node/subsystem-bench/src/lib/approval/mod.rs

```diff
@@ -785,6 +786,12 @@ fn build_overseer(
 	let db: polkadot_node_subsystem_util::database::kvdb_impl::DbAdapter<kvdb_memorydb::InMemory> =
 		polkadot_node_subsystem_util::database::kvdb_impl::DbAdapter::new(db, &[]);
 	let keystore = LocalKeystore::in_memory();
+	keystore
+		.sr25519_generate_new(
+			ASSIGNMENT_KEY_TYPE_ID,
+			Some(state.test_authorities.key_seeds.first().unwrap().as_str()),
```

Comment:
Nit: state.test_authorithies.key_seeds.get(NODE_UNDER_TEST) to make it future proof.